pkg/agent/config.go

```diff
@@ -30,6 +30,9 @@ const (
 	NimbessAgentPort = 9111
```

Comment:
is this something that would be in config?

Reply:
yeah it is. It's mapped to agent_port in the yaml config file.

Reply:
or were you trying to say you dont think it should be configurable?

Reply:
yeah was thinking maybe this could be in config. we can do that in a later patch. Things like these I just assumed we were picking values. Initially that is fine, but eventually we should move to config to avoid any collisions. If this is a well-known value then no need to make it configurable.

Reply:
it is configurable, 9111 is just the default value